src/main/java/org/sonarlint/intellij/ui/SonarLintAnalysisResultsPanel.java

```diff
@@ -51,6 +52,7 @@
   private transient AbstractScope scope;
 
   private transient ChangedFilesScope changedFilesScope;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "changedFilesScope" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1450)